e2e/playwright/support/utils.ts

```diff
@@ -0,0 +1,67 @@
+import path from 'node:path';
+
+import type {Page} from '@playwright/test';
+
+import {ChannelsPage} from '@e2e-support/ui/pages';
+import {UserProfile} from '@mattermost/types/users';
+import Client4 from '@mattermost/client/client4';
+
+const SCREENSHOTS_DIR = path.join(__dirname, '../screenshots');
+
+export const sleep = (millis = 500) => new Promise(r => setTimeout(r, millis));
```

Comment:
Shouldn't we use the native wait instead?

```suggestion
await page.waitForTimeout(3000);
```